packages/synapse-interface/store/reducer.ts

```diff
@@ -27,7 +27,7 @@ const persistedReducers = {
 export const storageKey: string = 'synapse-interface'
 
 export const persistConfig: PersistConfig<AppState> = {
-  version: 1, // upgrade to reset cache when updated data structures throw errors
+  version: 2, // upgrade to reset cache when updated data structures throw errors
```

Comment:
Do we need to update this here?

Reply:
Maybe not, added just in case since we're removing state from a persisted (transaction) slice. Can remove if you think it's unnecessary

Reply:
Checked this out and it looks like we don't have cache collision issues. Given we've removed all instances of the state both from reducers, functions, and around the app, there shouldn't be an opportunity for a crash.

Even if we don't upgrade persist version, users with an previous version of state won't have any code that would trigger a white screen issue.

The primary risk here is if we eventually end up adding back a transaction state value that shared the same names as the ones I just removed in this PR. To be on the safe side, we can still continue with upgrading persist version so we completely cache bust on all users. Only thing here is that we're going to wipe User's existing tx's when that happens